CTP/shell/src/com/navercorp/cubridqa/shell/main/TestFactory.java

```diff
@@ -83,7 +91,7 @@ private void addSkippedTestCases(ArrayList<String> list, String skippedType) {
 
 	public void execute() throws Exception {
 		
```

Comment:
Maybe we do not need isNeedContinue() method.
Just consider an existing method as below:
if (context.isContinueMode) {
    Dispatch.init(context);
    if(Dispatch.getInstance().getTotalTbdSize() == 0 ) {
        System.out.println("Not found any test cases to execute with CONTINUE mode");
        return;
    }
}